### PR TITLE
task 17

### PR DESCRIPTION
**Pull request by lesharadkovich**

### CODE REVIEW

File: tasks/17.txt [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+    },
+    { $unwind: "$categories_docs" },
+    {
+        $group: 
+        {
+            _id: "$categories_docs.CategoryName", 
+            AvgPrice: { $avg: "$UnitPrice" }
+        }
+    }
+])
```
Comment (ilya4321) [inline tasks/17.txt]: You can do the same with only one project after $lookup

**Comment by lesharadkovich** [comment]
But we still need to group, right?

**Comment by ilya4321** [comment]
Try to start from categories

**Comment by lesharadkovich** [comment]
Did it)

**Pull request closed without merging**